topics/ecology/tutorials/phylodiversity_workflow/tutorial.md

```diff
@@ -0,0 +1,200 @@
+objectives:
+- Learning how to use the phylodiversity workflow.
+- Compute endemism index
+- Create a phyloregion map
+time_estimation: ''
+tags:
+- ecology
+- taxonomic data
+contributors:
+- Morphy427
+
+---
+
+
+# Introduction
+
+This tutorial is designed to guide you through the Phylodiversity Galaxy workflow, demonstrating how to easily compute phylodiversity and create phyloregions from phylogeny, occupency and spatial files.
+
+The tutorial will provide a detailed explanation of inputs, workflow steps, and outputs. This tutorial gives a practical example, highlighting a use case extract from souhtern sea actinos populations.
+
+The primary goal of this workflow is to compute phylodiversity index and identify phyloregions. The project's objective is to offer accessible, reproducible and transparents solutions for analyse phylodiversity.
+
+This workflow is composed of four tools:
+
+- PhylOccuMatcher
+- CRSConverter
+- PhyloIndex
+- EstimEndem
+
+In this tutorial, we estimate your data are correctly formated.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Before starting
+
+This part will present the type of data you need to run the ecoregionalization workflow. This data will be downloaded in the next part of the tutorial.
+
+## phylogenic tree file
+The first file needed for this workflow is the phylogenetic tree of your interested species.
+In this example it'a a simplified phylogeny of the actinopterigy
+This file must be at newick format.
+
+## occupancy file
+The second file is an occupancy file, each line is a species, the decimal separator must be "." and the column must be separated with "\t"(={Tabulation}). You need to have a column "grids" containing the cell of the grid you've seen your species and the name of the column with the species names must be "newscientificname".
+
+| grids   | newscientificname   | ...     | ...   | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+| ------- | ------------------- | ------- | ----- | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+| ...     | ...                 | ...     | ...   | ...   |
+| ------- | ------------------- | ------- | ----- | ----- |
+
+## Shapefile
+The last file is a spatial file in shapefile format. In Galaxy this type of file must be uploaded as a composite file of type shp.
+This kind of file must have at least 3 file with the same name and 3 different extension : .shp, .shx end .dbf. you can have more file optionally like the .prj file.
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/api/records/15487657/files/shapefile.dbf/content
```

Comment:
we can rewrite the Zenodo links a bit:

```suggestion
>    https://zenodo.org/records/15487657/files/shapefile.dbf
```
(and do the same for other links)

Reply:
yes, thank you!